Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.Collections;
+using System.Diagnostics;
+using PatchKit.Unity.Patcher;
+using PatchKit.Unity.Patcher.Debug;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace PatchKit.Unity.Patcher.Debug
+{
+    public class DebugMenu : MonoBehaviour
+    {
+        private static readonly DebugLogger DebugLogger = new DebugLogger(typeof(DebugMenu));
+
+        private Rect _popupRect, _rect;
+        private bool _showDebugMenu;
+        private bool _showPopup;
+        private string _popupMessage;
+        private GraphicRaycaster _graphicRaycaster;
+
+        void Start()
+        {
+            int windowWidth = 250;
+            int windowHeight = 180;
+            int x = (Screen.width - windowWidth) / 2;
+            int y = (Screen.height - windowHeight) / 2;
+            int popupRectY = (Screen.height - 120) / 2;
+            _rect = new Rect(x, y, windowWidth, windowHeight);
+            _popupRect = new Rect(x, popupRectY, windowWidth, 120);
+            _graphicRaycaster = FindObjectOfType<GraphicRaycaster>();
+        }
+
+        void OnGUI()
+        {
+            if (Event.current.Equals(Event.KeyboardEvent(KeyCode.D.ToString())))
```

Comment:
Is it possible to change the shortcut to something like "Ctrl+Shift+D"?

Just to make it less possible for someone to accidentally open the debug menu.

Reply:
done